CHANGES.rst

```diff
@@ -2,8 +2,12 @@
  Changes
 =========
 
-5.0 (unreleased)
-================
+4.9.4 (unreleased)
```

Comment:
I think the NFC change was considered potentially breaking, so a major version bump to 5.0.0 was appropriate.